src/Inspection.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+
+namespace RicorocksDigitalAgency\Morpher;
+
+
+class Inspection
+{
+    public $beforeMigratingHooks;
+    public $beforeHooks;
+    public $afterHooks;
+
+    public function __construct()
+    {
+        $this->beforeMigratingHooks = collect();
+        $this->beforeHooks = collect();
+        $this->afterHooks = collect();
+    }
+
+    public function beforeMigrating(callable $closure)
```

Comment:
I wonder if this could be `beforeThisMigrates` or something along those lines? To clarify that it's not a global `beforeMigrating`, but local to this Morph?

Reply:
I've renamed to `beforeThisMigration`